lesson5/public/app.js

```diff
@@ -0,0 +1,59 @@
+const app = new Vue({
+    el: '#app',
+    data: {
+        items: [],
+        cart: [],
+        searchQuery: '',
+        display: 'none',
+        quantity: 0,
+        putInCart: 0
+    },
+    methods: {
+        putInCart: function(event) {
+            if (event.target.tagName === 'BUTTON') {
+                fetch('http://localhost:3000/cart', {
+                    method: 'POST',
+                    body: JSON.stringify(item),
+                }).then((cart) => {
+                    this.cart.push({item
```

Comment:
скобочки наверное не нужны)
```suggestion
                    this.cart.push(item
```